README.md

```diff
@@ -160,8 +176,9 @@ Our documentation features [a collection of example projects using Material UI](
 
 ## Documentation
 
-- [Material UI](https://mui.com/material-ui/getting-started/installation/)
-- [MUI Base](https://mui.com/base/getting-started/installation/)
+- [Material UI](https://mui.com/material-ui/getting-started/overview/)
+- [Joy UI](https://mui.com/base/getting-started/overview/)
```

Comment:
```suggestion
- [Joy UI](https://mui.com/joy-ui/getting-started/overview/)
```

Reply:
Whoops! Forgot to clean up the copypasta. 😅